packages/components/organisms/f-checkout/stories/Checkout.stories.js

```diff
@@ -39,11 +40,23 @@ CheckoutMock.setupCheckoutMethod(getBasketDeliveryUrl);
 CheckoutMock.setupCheckoutMethod(getBasketCollectionUrl);
 CheckoutMock.setupCheckoutMethod(updateCheckoutUrl);
 CheckoutMock.setupCheckoutMethod(updateCheckoutErrorsUrl);
+CheckoutMock.setupCheckoutMethod(updateCheckoutServerErrorUrl);
 CheckoutMock.setupCheckoutMethod(getAddressUrl);
 CheckoutMock.setupCheckoutMethod(placeOrderUrl);
 CheckoutMock.setupCheckoutMethod(getGeoLocationUrl);
 CheckoutMock.passThroughAny();
 
+const checkoutIssues = 'Checkout Issues (response from server but order not fulfillable)';
+const serverError = 'Server Error (Response from server is unsuccessful HTTP error code)';
```

Comment:
Nit: could you please ensure we stick to the same pattern of casing and wording (i.e. `response` vs `Response` and `unsuccessful` vs nothing saying `successful`, hehe).

Reply:
Done.